lib/matplotlib/widgets.py

```diff
@@ -1447,6 +1447,77 @@ def _update(self):
             self.canvas.draw_idle()
 
 
+class MultiAxesCrossCursor(MultiCursor):
+    """
+    Provide a vertical (default)  and/or a horizontal line cursor shared
+    between multiple axes.
+
+    Tt's similar to MultiCursor,but only one horizontal line in figure.
+
+    For the cursor to remain responsive you must keep a reference to
+    it.
+
+    Example usage::
+
+        from matplotlib.widgets import MultiAxesCrossCursor
```

Comment:
Can you make this example not depend on pylab? Otherwise this looks good to me.

Reply:
This cursor and example is based on MultiCursor,the only difference is horizontal line.

Reply:
I have changed example not depend on pylab yet.